### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sonukumar7949**
## Checks

- [x] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) and Verified by Github.
- [x] All commits in this Pull Request follow the Ockam [commit message convention](https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md#commit-messages).
- [x] I accept the Ockam Community [Code of Conduct](https://github.com/build-trust/.github/blob/main/CODE_OF_CONDUCT.md).
- [x] I have accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/blob/main/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam-contributors](https://github.com/build-trust/ockam-contributors) repository.

### CODE REVIEW

**Comment by sonukumar7949** [comment]
update readme.md file

**Comment by sonukumar7949** [comment]
hello sir can i solve this repo

File: README.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ After the binary downloads, please move it to a location in your shell's `$PATH`
 Let's build a solution for a very common topology. A application service and an application client running
 in two private networks wish to communicate with each other without exposing ports on the Internet.
 
-```bash
+``bash
```
Comment (twittner) [inline README.md:72]: The triple backticks introduce a code block (see https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks) and AFAICS the syntax was correct.

**Review by twittner — COMMENTED** [review]
Thank you for opening a PR. What was the motivation for this change? As explained the triple backticks are valid syntax for code blocks.

File: README.md (hunk 1 above)
Comment (sonukumar7949) [reply README.md:72]: Sorry sir actually I forget it & can I resolve.

**Comment by mrinalwadhwa** [comment]
@sonukumar7949 closing this one. If you want try another issue please have a look at our list of good first issues https://github.com/build-trust/ockam/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22

**Pull request closed without merging**